core/services/directrequest/delegate.go

```diff
@@ -223,7 +225,7 @@ func (*listener) JobID() models.JobID {
 
 // Job complies with log.Listener
 func (l *listener) JobIDV2() int32 {
-	return l.spec.ID
+	return l.job.ID
```

Comment:
Is it possible to get a regression test?

Reply:
good point

Reply:
@samsondav added - and i checked that it fails under the previous code